src/csharp/Microsoft.Spark.E2ETest/IpcTests/ML/Feature/BucketizerTests.cs

```diff
@@ -25,7 +25,8 @@ public BucketizerTests(SparkFixture fixture)
         [Fact]
         public void TestBucketizer()
         {
-            var expectedSplits = new double[] { double.MinValue, 0.0, 10.0, 50.0, double.MaxValue };
+            var expectedSplits =
+                new double[] { double.MinValue, 0.0, 10.0, 50.0, double.MaxValue };
 
             string expectedHandle = "skip";
             string expectedUid = "uid";
```

Comment:
nit: can we make this strings consts?

Reply:
@Niharikadutta any particular reason you wanted these as consts ?

Reply:
Only because they were not being modified in the scope.

Reply:
@GoEddie Let's revert this change for consistency. Apologize for the confusion!

Reply:
reverted!